mavis/illustrate/scatter.py

```diff
@@ -86,22 +81,24 @@ def __init__(
         self.ymax = ymax
         self.points = points
         if self.ymin is None and (yticks or points):
-            self.ymin = min([y.start for x, y in points] + yticks)
+            self.ymin = min([y for x, y in points] + yticks)
         if self.ymax is None and (yticks or points):
-            self.ymax = max([y.end for x, y in points] + yticks)
+            self.ymax = max([y for x, y in points] + yticks)
         self.xmin = xmin
         self.xmax = xmax
         if self.xmin is None and points:
-            self.xmin = min([x.start for x, y in points])
+            self.xmin = min([x for x, y in points])
         if self.xmax is None and points:
-            self.xmax = max([x.end for x, y in points])
+            self.xmax = max([x for x, y in points])
         self.y_axis_label = y_axis_label
         self.height = 100
```

Comment:
shouldn't self.height = height and self.point_radius = point_radius

Reply:
yup, I will fix that